packages/nuxt/src/utils/styles/optimize-css.ts

```
@@ -1,16 +1,8 @@
-    code: Buffer.from(css),
-    minify: true,
-    sourceMap: true,
-  });
-
-  return res.code.toString();
-  return css;
+  return css
+    .replace(/\/\*[\s\S]*?\*\//g, '')
+    .replace(/\s+/g, ' ')
+    .replace(/\s*([{}:;,])\s*/g, '$1')
+    .replace(/;}/g, '}')
```

ESLint

[error] Unescaped source character '}'.

(regexp/strict)